include/discover_poco.php

```diff
@@ -36,11 +36,14 @@ function discover_poco_run(&$argv, &$argc){
 		$mode = 2;
 	} elseif(($argc == 2) && ($argv[1] == "suggestions")) {
 		$mode = 3;
+	} elseif(($argc == 3) && ($argv[1] == "server")) {
+		$mode = 4;
```

Comment:
Can you please add a doc block for this function detailing the different mode and the expected parameters?